graph/graph_test.go

```diff
@@ -301,6 +301,6 @@ func tempGraph(t *testing.T) (*Graph, graphdriver.Driver) {
 }
 
 func nukeGraph(graph *Graph) {
```

Comment:
`func (g *Graph) nuke()`

Do we need to export the `Root` property? I feel like that should be hidden.